services/manifest-repo-export-service/pkg/repository/repository.go

```diff
@@ -566,9 +566,17 @@ func (r *repository) ApplyTransformers(ctx context.Context, transaction *sql.Tx,
 	}
 
 	// TODO this will be handled in Ref SRX-PA568W
+	authorEmail, ok := (*transformers[0].GetMetadata())["authorEmail"]
+	if !ok {
+		authorEmail = "invalid@gmail.com"
+	}
+	authorName, ok := (*transformers[0].GetMetadata())["authorName"]
```

Comment:
nitpick: I would prefer to call `GetMetadata()` once and store it in a variable.